js/mail.js

```diff
@@ -587,6 +591,10 @@ var Mail = {
 
 			// focus 'to' field automatically on clicking New message button
 			$('#to').focus();
+			if (!_.isUndefined($(data.currentTarget).data().email)) {
```

Comment:
if I open a mail in the **drafts** folder this happens:
`Uncaught TypeError: Cannot read property 'email' of null`

Seems like `data.currentTarget` doesn't exists there and the desired information are in data itself.

@DeepDiver1975